fms2_io/include/gather_data_bc.inc

```diff
@@ -0,0 +1,256 @@
+!* This file is part of the GFDL Flexible Modeling System (FMS).
```

Comment:
The top of the license header is missing?

Reply:
Fixed in f0d7161f31478f7ef7f92b4eee505d409ef3b0eb